app/src/main/java/com/android/sample/model/profile/User.kt

```diff
@@ -24,3 +31,12 @@ val InterestCategories =
         "Cultural Activity",
         "Art Activity",
         "Musical Activity") + "None"
+val InterestCategoriesColors =
+    mapOf(
+        "Sport" to SPORT_COLOR,
+        "Outdoor Activity" to OUTDOOR_COLOR,
+        "Indoor Activity" to INDOOR_COLOR,
+        "Cultural Activity" to CULTURAL_ACTIVITY_COLOR,
+        "Art Activity" to ART_ACTIVITY_COLOR,
+        "Musical Activity" to MUSICAL_ACTIVITY_COLOR,
+        "None" to Color.Gray)
```

Comment:
These will be reduced to the categories used in activities in a next PR

Reply:
Yes exactly !